asv_bench/benchmarks/omnisci/benchmarks.py

```diff
@@ -47,7 +46,7 @@ class TimeJoin:
     def setup(self, shape, how):
         self.df1 = generate_dataframe(ASV_USE_IMPL, "int", *shape, RAND_LOW, RAND_HIGH)
         self.df2 = generate_dataframe(ASV_USE_IMPL, "int", *shape, RAND_LOW, RAND_HIGH)
-        trigger_import(self.df1, self.df2)
+        execute([self.df1, self.df2], trigger_omnisci_import=True)
```

Comment:
Please do not change the rest of the benchmarks, as this will lead to not displaying previous results in dashboards.

I propose to transfer `trigger_import` to `asv_bench/benchmarks/utils.py` and use it in `execute` function.

Something like:
```python
    if trigger_omnisci_import:
        trigger_import(dfs)
        return
    if ASV_USE_IMPL == "modin":
        if ASV_USE_BACKEND == "omnisci":
            df._query_compiler._modin_frame._execute()
            return
        partitions = df._query_compiler._modin_frame._partitions
        all(
            map(
                lambda partition: partition.drain_call_queue() or True,
                partitions.flatten(),
```

Reply:
Ok, i see, your suggestion was implemented.